Source/Core/DolphinQt/Settings/AdvancedPane.cpp

```diff
@@ -68,6 +68,12 @@ void AdvancedPane::CreateLayout()
       "Enables the Memory Management Unit, needed for some games. (ON = Compatible, OFF = Fast)"));
   cpu_options_group_layout->addWidget(m_enable_mmu_checkbox);
 
+  m_pause_on_panic_checkbox = new QCheckBox(tr("Pause on Panic"));
+  m_pause_on_panic_checkbox->setToolTip(
+      tr("Pauses the emulation if a Read/Write or Unknown Instruction panic occurs.\nEnabling will "
```

Comment:
This description is misleading, I came across invalid read/write from `WriteToHardware` and `ReadToHardware` with the associated `Unable to resolve read/write address` panic alerts and it didn't break the execution. You should probably reword it to emphasis that it breaks on unknown instructions and DSI exceptions instead.

Reply:
Ideally, I would like to extend this to all of the write/read exceptions. Is there specific areas I should look into for this?

Reply:
The invalid read/writes (they are not exceptions) that sepalani is referring to are these:

https://github.com/dolphin-emu/dolphin/blob/349c484ec803ccba96bd334da5de771cfd107029/Source/Core/Core/PowerPC/MMU.cpp#L246
https://github.com/dolphin-emu/dolphin/blob/349c484ec803ccba96bd334da5de771cfd107029/Source/Core/Core/PowerPC/MMU.cpp#L400

Reply:
Added these. As a caveat, these continue to re-appear when pressing "OK".
If picking "Ignore" it works as intended. The "Reset Panic Handler" drop down from Tools can be used in conjunction as a work around.